### PR TITLE
Update inline volume spec comments

### PR DESCRIPTION
**Pull request by Jiawei0227**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
CSI Migration feature has been beta since v1.17. Change the comment to reflect that.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

/sig storage
/assign @msau42

### CODE REVIEW

File: pkg/apis/storage/types.go [hunk 1]
```diff
@@ -158,7 +158,7 @@ type VolumeAttachmentSource struct {
 	// a persistent volume defined by a pod's inline VolumeSource. This field
 	// is populated only for the CSIMigration feature. It contains
 	// translated fields from a pod's inline VolumeSource to a
-	// PersistentVolumeSpec. This field is alpha-level and is only
+	// PersistentVolumeSpec. This field is beta-level and is only
```
Comment (msau42) [inline pkg/apis/storage/types.go:161]: I think you also need to regenerate the openapi specs. See https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md#generate-code

Comment (msau42) [inline pkg/apis/storage/types.go:161]: The versioned types.go (v1beta, v1) also need to be updated

Comment (Jiawei0227) [reply pkg/apis/storage/types.go:161]: Done

Comment (Jiawei0227) [reply pkg/apis/storage/types.go:161]: Done. I have always been curious what's the relationship between staging/apis and pkg/apis

Comment (Jiawei0227) [reply pkg/apis/storage/types.go:161]: Okay, it is all addressed in the doc: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md#generate-code

**Pull request merged**